railties/lib/rails/commands/credentials/credentials_command.rb

```diff
@@ -49,6 +51,7 @@ def show
       option :disenroll, type: :boolean, default: false,
         desc: "Disenrolls project from credentials file diffing"
 
+      desc "diff", "Enrolls/disenrolls project in credentials file diffing with `git diff`"
```

Comment:
I think we should say "decrypted" somewhere in here.  Maybe:

```suggestion
      desc "diff", "Enrolls/disenrolls credentials in decrypted diffing using git"
```

Or:

```suggestion
      desc "diff", "Enrolls/disenrolls in decrypted diffs of credentials using git"
```

?